OpenRA.Game/Network/Session.cs

```diff
@@ -185,7 +185,7 @@ public class Global
 			public bool Fog = true;
 			public bool AllyBuildRadius = true;
 			public int StartingCash = 5000;
-			public string TechLevel = "none";
+			public string TechLevel = "Unrestricted";
```

Comment:
This doesn't have anything to do with extracting chat content from replay files, so please move it to another pr.